src/assets/scripts/client/App.js

```diff
@@ -100,9 +84,11 @@ export default class App {
             $.getJSON('assets/aircraft/aircraft.json')
         )
             .done((airportResponse, airlineResponse, aircraftResponse) => {
+                const initialAirport = typeof airportResponse[0] !== 'undefined' ? airportResponse[0] : null;
```

Comment:
oh I like this!  I'm spinning up a review app to check this out.  But I think this could be a winner.

small style gripe:
```javascript
// instead of
const initialAirport = typeof airportResponse[0] !== 'undefined' ? airportResponse[0] : null;

// please use
const initialAirport = typeof airportResponse[0] !== 'undefined' 
    ? airportResponse[0] 
    : null;
```

if just makes the ternaries a little easier to read.   I'm not a huge fan of them to begin with, but they are useful in cases such as this.